service/server.go

```diff
@@ -47,6 +49,15 @@ func serve() {
 	router.HandleFunc("/health", Health).Methods("GET")
 	router.HandleFunc("/", Up).Methods("GET", "POST")
 
+	//Creates an LRU cache of the given size
+	var err error
+	db.LruCache, err = lru.New(5)
+	if err != nil {
+		log.Errorf("err: %v", err)
+	}
+	log.Infoln("LRU cache created with given size")
+	db.InitLruCacheDBSchema = false
```

Comment:
@Lucy911, does it make sense to set this bool here in `server.go` or would there be a way to tie that into the construction of the cache which happens thru `lru.New(5)`? seems like something that would otherwise be easy for callers to forget to do and is worth abstracting away, if feasible

Reply:
the boolean variable default value is false, if you don't want to set it to true, simply remove this assignment

Reply:
Thanks for the comments. Just made the changes accordingly.